### ISSUE TITLE
linkedin announced the breaking change in their developer program, will that affect the library?

### ISSUE THREAD

**Issue by boris-petrov1214** [auth0/passport-linkedin-oauth2#29]

**Comment by xeroxoid** [comment]
It seems that it has...

**Comment by xeroxoid** [comment]
So to get your app to work you have to limit your scope to the now allowed permissions. i.e.:

````javascript
  ...
  linkedinStrategy = new LinkedInStrategy({
      clientID: config.linkedin.clientID,
      clientSecret: config.linkedin.clientSecret,
      callbackURL: config.linkedin.callbackURL,
      scope: ['r_basicprofile', 'w_share', 'r_emailaddress' ],
      passReqToCallback: true
    },...
  ...
````

So no more leeching on them profiles :)

**Comment by sidonaldson** [comment]
Yup, so "r_contactinfo" is now allowed only for authorised partners. I guess they want you to still use linkedin ;)

**Comment by chewedon** [comment]
I'm using Sailsjs. 

The call to: 

    login: function(req, res) {		
		passport.authenticate('linkedin', function(err, user, info) {
			// The request will be redirected to LinkedIn for authentication, so this 
    		        // function will not be called. 
		});
    }

Does not redirect the user to the Linkedin login prompt for the user to enter email and password, could it be related to Linkedin's server changes or am I using this library in a wrong way ?

At the moment, my browser just hangs there, spinning and never completing the request for an access token.
